components/src/components/TextInput/TextInput.styles.tsx

```diff
@@ -29,7 +29,7 @@ export const StyledIcon = styled(Icon)<StyledIconProps>`
           top: ${`calc(50% - ${iconSizes.DdsIconsizeMediumNumberPx / 2}px)`}; ;
         `}
   `}
-  z-index: 1;
+  z-index: 10;
```

Comment:
Trenger denne å være større enn 1? Skal vel ikke legge seg over noe annet enn `TextInput`?

Reply:
Tenkte hvis det skulle dukke op noe er det greit å ha muligheten, men jeg kommer ikke på eksempler. Men du tenker det er unødvendig?

Reply:
Jeg ville typisk ha brukt verdier mellom 1-10 på elementer som egentlig ikke har en høyere prioritet globalt, men som man ønsker å plassere på en bestemt måte i forhold til hverandre. 

Ser for meg at man lettere får en tilstand der man har noe over `TextInput`, men ikonet vil legge seg øverst. Ikke noe stor sak, men tenker at det heller ikke er nødvendig ha et gap i `z-index` - dukker det opp et behov kan man håndtere det da 😄